src/FlaUI.Core.UnitTests/Logging/AbstractBaseLoggerTests.cs

```diff
@@ -0,0 +1,300 @@
+﻿using FlaUI.Core.Logging;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Globalization;
+
+namespace FlaUI.Core.UnitTests.Logging
```

Comment:
Can we change the default namespace on the unit test assembly to match the default namespace of the assembly under test?